heron/tools/tracker/src/python/routers/metrics.py

```diff
@@ -151,19 +152,19 @@ class TimelinePoint(BaseModel): # pylint: disable=too-few-public-methods
 
 class MetricsQueryResponse(BaseModel): # pylint: disable=too-few-public-methods
   """A metrics timeline over an interval."""
-  start_time: int = Field(..., alias="starttime")
-  end_time: int = Field(..., alias="endtime")
+  starttime: int = Field(..., alias="starttime")
+  endtime: int = Field(..., alias="endtime")
   timeline: List[TimelinePoint] = Field(
       ..., description="list of timeline point objects",
   )
 
 
-@router.get("/metricsquery", response_model=MetricsQueryResponse)
+@router.get("/metrics/query", response_model=MetricsQueryResponse)
 async def get_metrics_query( # pylint: disable=too-many-arguments
     cluster: str,
-    role: Optional[str],
     environ: str,
     query: str,
+    role: Optional[str] = None,
     start_time: int = Query(..., alias="starttime"),
     end_time: int = Query(..., alias="endtime"),
```

Comment:
If you switch over to `str` in the class definition these will need to be updated as well.